### PR TITLE
fix for org= always in first place

### PR DESCRIPTION
**Pull request by MT255026**
QueryBand                                        |
-------------------------------------------------+
=S> org=teradata-internal-telem;appname=fivetran;|
=S> org=teradata-internal-telem;appname=fivetran;|
                                                 |
=S> org=teradata-internal-telem;appname=fivetran;|
=S> org=teradata-internal-telem;appname=fivetran;|

### CODE REVIEW

File: src/test/java/com/teradata/fivetran/destination/HandleQueryBandTest.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    void appnameWithUpperCaseFivetran_shouldNotAppend() {
+        String result = TeradataJDBCUtil.handleQueryBand("appname=FIVETRAN_APP;");
+        assertEquals(
+                "org=teradata-internal-telem;appname=FIVETRAN_APP;",
+                result
+        );
+    }
+
+    @Test
+    void orgAtEnd_shouldBeMovedToFirst() {
```
Comment (sc250072) [inline src/test/java/com/teradata/fivetran/destination/HandleQueryBandTest.java]: Add tests for null and empty cases

File: src/main/java/com/teradata/fivetran/destination/TeradataJDBCUtil.java [hunk 2]
```diff
@@ -112,51 +112,64 @@ private static String[] writeSslCertToFile(String sslCert) throws IOException {
...
+        );
 
-        // Check if "appname" key exists
-        if (queryBandMap.containsKey("appname")) {
-            // Check if "airflow" exists in the value of "appname" key
-            String appnameValue = queryBandMap.get("appname").toLowerCase();
-            if (!appnameValue.contains("fivetran")) {
-                queryBandMap.put("appname", queryBandMap.get("appname") + "_fivetran;");
+        // ✅ Handle appname SECOND
+        if (parsed.containsKey("appname")) {
```
Comment (sc250072) [inline src/main/java/com/teradata/fivetran/destination/TeradataJDBCUtil.java]: containsKey is also case sensitive. It wont be true for words "APPNAME", "Appname"

**Review by sc250072 — CHANGES_REQUESTED** [review]
Couple of review comments addressed

File: src/main/java/com/teradata/fivetran/destination/TeradataJDBCUtil.java [hunk 3]
```diff
@@ -112,51 +112,64 @@ private static String[] writeSslCertToFile(String sslCert) throws IOException {
...
-        }
+        // ✅ Always put org FIRST
+        queryBandMap.put(
+                "org",
+                parsed.getOrDefault("org", "teradata-internal-telem")
+        );
 
-        // Check if "appname" key exists
-        if (queryBandMap.containsKey("appname")) {
-            // Check if "airflow" exists in the value of "appname" key
```
[the remark this answers is not in the document]
Comment (MT255026) [reply src/main/java/com/teradata/fivetran/destination/TeradataJDBCUtil.java]: Got this code from the airflow. Anyway, removed now.

File: src/main/java/com/teradata/fivetran/destination/TeradataJDBCUtil.java [hunk 4]
```diff
@@ -112,51 +112,64 @@ private static String[] writeSslCertToFile(String sslCert) throws IOException {
...
             }
+            queryBandMap.put("appname", appnameValue);
         } else {
-            // Add "appname=fivetran;" if "appname" key does not exist
-            queryBandMap.put("appname", "fivetran;");
+            queryBandMap.put("appname", "fivetran");
         }
 
-        // Reconstruct the queryBand string from the map
+        // ✅ Add remaining keys (if any), preserving original behavior
```
[the remark this answers is not in the document]
Comment (MT255026) [reply src/main/java/com/teradata/fivetran/destination/TeradataJDBCUtil.java]: Removed

**Review by vs255034 — APPROVED** [review]
Looks Good To Me
